faces/wildfly-mods/transform.xslt

```diff
@@ -13,6 +13,8 @@
                          //*[local-name()='profiles']">
         <xsl:copy>
             <xsl:apply-templates select="document('profile.xml')/*"/>
+            <xsl:apply-templates select="document('global-excludes-profile.xml')/*"/>
+            <xsl:apply-templates select="document('se21-excludes-profile.xml')/*"/>
```

Comment:
Note: I tried to include 3 'profile' elements in profile.xml and it didn't work. When my time budget for renewing my xslt skills was used up I went with 3 separate files.